packages/framework/src/validators/validator.test.ts

```diff
@@ -40,7 +40,7 @@ describe('validators', () => {
         title: 'should remove additional properties and successfully validate',
         schemas: {
           zod: z.object({ name: z.string() }),
-          json: { type: 'object', properties: { name: { type: 'string' } } } as const,
+          json: { type: 'object', properties: { name: { type: 'string' } }, additionalProperties: false } as const,
```

Comment:
Necessary to fix the failing test after modifying the `removeAdditional` property in the JsonSchema validator.